gulpfile.js

```diff
@@ -241,8 +241,25 @@ function endBuildStep(stepName, targetName, startTime) {
  * @return {!Promise}
  */
 function buildExtensions(options) {
-  var results = [];
+  const results = [];
```

Comment:
This shouldn't be a const, since you're editing it below. Revert this line.

Reply:
reverted